test/models/work_test.rb

```diff
@@ -0,0 +1,115 @@
+require "test_helper"
+require 'pry'
+
+describe Work do
+  describe 'relations' do
+    it 'has many votes' do
+      work = works(:nemo)
+
+      work.votes.length.must_equal 3
+      work.votes[0].must_equal votes(:one)
+    end
+  end
+
+  describe 'validations' do
+    describe 'title' do
+      it 'is valid when title is present' do
+        is_valid = works(:nemo).valid?
+        expect( is_valid ).must_equal true
+      end
+
+      it 'is invalid without a title' do
+        w = Work.new(
+          category: 'movie',
+          title: nil,
+          creator: 'Pixar',
+          pub_year: 2001,
+          description: 'daddy issues',
+        )
+        is_valid = w.valid?
+
+        expect( is_valid ).must_equal false
+        expect( w.errors.messages ).must_include :title
+      end
+
+      it 'is invalid with a non-unique title' do
+        repeat = Work.new(
+          category: 'movie',
+          title: 'Finding Nemo',
```

Comment:
it might be nice to re-inforce the point of this test by referencing the title of an existing work (defined in fixtures), aka
```ruby
title: works(:nemo).title,
```